src/openrct2/actions/LandSetHeightAction.cpp

```diff
@@ -246,7 +246,7 @@ money32 LandSetHeightAction::GetSmallSceneryRemovalCost() const
         if (sceneryEntry == nullptr)
             continue;
 
-        cost += MONEY(sceneryEntry->removal_price, 0);
```

Comment:
For cases like this, I would still use the macro. It is clearer.

Reply:
I think the macro is removed (?)

Not sure though, will check later

Reply:
Yes, it's removed.

Reply:
Then I think it should be reintroduced, for cases like these.

Reply:
What about some `constexpr` function like:

```c++
constexpr money32 toGBP(long double money) noexcept
{
    return money * 10;
}
```

At least we dodge from macros :P. I noticed most of the places that are not using literals are either using the macro with a fraction equal to 0, or a double/float already.


PS: I think while searching for this I also found some places where the conversion was not right after this patch, see the latest commit.

Reply:
I went ahead and added it anyway to speed things up.

Reply:
Code should only ever use floating point for ui. Can you restructure this so that it follows that convention.

Reply:
Not sure what you meant, some of the parameters are `double`, like: 
```c++
            money32 money = toGBP(double_val[0]);
```

Reply:
yeah but double_val[0] is from the interactive console which is ui. I know it would never cause issues but I like the convention of no floating point makes things easy to reason about and reduces the chance of difficult to think about issues.

Reply:
Sorry, still not sure about what you're proposing.

Reply:
I think that's the only missing piece from this PR now @duncanspumpkin